org.robocode.Robocode.yml

```diff
@@ -0,0 +1,93 @@
+app-id: org.robocode.Robocode
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: robocode
+rename-appdata-file: robocode.appdata.xml
+rename-desktop-file: robocode.desktop
+rename-icon: robocode
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
```

Comment:
All dconf permissions are not needed. DConf will user app-specific keyfile to store settings.